librz/core/cmd/cmd_analysis.c

```diff
@@ -430,6 +430,31 @@ static const char *help_msg_as[] = {
 	NULL
 };
 
+/**
+ * \brief Helper to get function in \p offset
+ *
+ * Case of overlapped functions is treated as an error.
+ */
+static RzAnalysisFunction *analysis_get_function_in(RzAnalysis *analysis, ut64 offset) {
```

Comment:
Please name to `analysis_get_first_function_in()`

Reply:
Right above you can see `Case of overlapped functions is treated as an error`. So this function returns `NULL` if there are more than one function in offset.